package.json

```diff
@@ -24,11 +22,6 @@
   },
   "homepage": "https://github.com/airtap/browsers",
   "devDependencies": {
-    "babel": "^5.8.29",
-    "babel-eslint": "^4.1.3",
-    "eslint": "^1.7.3",
-    "eslint-config-airbnb": "^0.1.0",
-    "eslint-config-algolia": "^4.2.0",
-    "eslint-plugin-react": "^3.6.3"
+    "babel": "^5.8.29"
```

Comment:
Why is `babel` even needed?

Reply:
It isn't, so I removed it.

Reply:
(it was only there for the use of ES6 modules)